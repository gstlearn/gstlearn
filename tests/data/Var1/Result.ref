
Variogram characteristics
=========================
Number of variable(s)       = 1
Number of direction(s)      = 1
Space dimension             = 2
Variable(s)                 = [Simu]
Variance-Covariance Matrix     0.616

Direction #1
------------
Number of lags              = 20
Direction coefficients      =      1.000     0.000
Direction angles (degrees)  =      0.000
Tolerance on direction      =      0.000 (degrees)
Calculation lag             =      0.010
Tolerance on distance       =     50.000 (Percent of the lag value)
Grid Direction coefficients =          1         0

For variable 1
      Rank    Npairs  Distance     Value
         1  9900.000     0.010     0.234
         2  9800.000     0.020     0.258
         3  9700.000     0.030     0.286
         4  9600.000     0.040     0.309
         5  9500.000     0.050     0.340
         6  9400.000     0.060     0.366
         7  9300.000     0.070     0.397
         8  9200.000     0.080     0.426
         9  9100.000     0.090     0.456
        10  9000.000     0.100     0.481
        11  8900.000     0.110     0.504
        12  8800.000     0.120     0.527
        13  8700.000     0.130     0.553
        14  8600.000     0.140     0.563
        15  8500.000     0.150     0.574
        16  8400.000     0.160     0.590
        17  8300.000     0.170     0.605
        18  8200.000     0.180     0.610
        19  8100.000     0.190     0.616

Model characteristics
=====================
Space dimension              = 2
Number of variable(s)        = 1
Number of basic structure(s) = 2
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Nugget Effect
- Sill         =      0.205
Spherical
- Sill         =      0.496
- Range        =      0.268
Total Sill     =      0.701
Known Mean(s)     0.000
Goodness-of-fit (as a percentage of the variance) corresponds to level #1 (1 for very good)
