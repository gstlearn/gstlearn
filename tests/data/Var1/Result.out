
Optimization Conditions
=======================
- Number of variables       1  
- Space dimension           2  
- Number of experiments     19  
- Number of parameters      1  
- Constrained Minimization  OFF
- Intrinsic option          OFF
- Anisotropy                OFF
- Anisotropy Rotation       OFF
- Global Rotation           OFF
- Rotation around Z only    OFF
- Lock third dimension      OFF
- Lock 2-D Isotropy         OFF
- Keep Intrinsic structure  OFF
- Use the Goulard option    ON
- Keep all structures       OFF

Model
-----
Structure : Spherical
 1 - Range U : 0.190668 [0.000000,NA[

Trajectory of parameters in Foxleg Algorithm
--------------------------------------------
 Iteration     Score     Delta     Par-1
         1  9.71e-07         1     0.191
         2  8.14e-08         1     0.201
         3  6.97e-08         1     0.203

Model
-----
Structure : Spherical
 1 - Range U : 0.202962 [0.000000,NA[

Model characteristics
=====================
Space dimension              = 2
Number of variable(s)        = 1
Number of basic structure(s) = 2
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Nugget Effect
- Sill         =      0.201
Spherical
- Sill         =      0.400
- Range        =      0.203
Total Sill     =      0.600
